lib/+wbs/@Contacts/Contacts.m

```diff
@@ -234,30 +258,43 @@
         end
 
         function prepare_optimization_matrix(obj)
-            % prepare_optimization_matrix Fills the matrix used in the optimization problem
-
+            % prepare_optimization_matrix Fills the matrix used by the optimization problem solver.
+            % A being the matrix of friction cone constraints, for ...
+            % - quadprog: Ax <= b (should include -x <= 0).
+            % - OSQP    : l <= Ax <= u (should include -Inf <= -x <= 0).
+            %
+            % So in both cases we define: AxLb <= Ax <= AxUb where AxLb = -Inf.
+            
             total_num_vertices = obj.num_vertices * 2; % number of vertex per foot * number feet
-            num_variables = 3 * total_num_vertices; % number of unknowns - 3 forces per vertex
+            num_variables = 3 * total_num_vertices; % number of unknowns (3 force components per vertex)
             num_constr = 5 * total_num_vertices; % number of constraint: simplified friction cone + non negativity of vertical force
             % fill the optimization matrix
             obj.A = zeros(num_constr, num_variables);
-            obj.b = zeros(num_constr, 1);
+            obj.AxUb = zeros(num_constr, 1);
```

Comment:
This name could be misleading (I am referring to AxUb),  is it the upper boud right? like this could seem A*x*b

Reply:
I actually think that `UB` or `ub` or just `u` as often used, is misleading, as it can be the boundary for `x` or for the product $A x$. And `Aub` is also misleading. That is why I preferred `AxUb` as the boundary of the product `Ax`. Note that I clarify the use of the variable in the comment on top of that paragraph. If it's really disturbing, I can rename it `Ax_Ub`. Would that be better?

Reply:
Renamed as agreed.